tests/e2e/api-interface.spec.ts

```diff
@@ -79,10 +96,11 @@ describe('The API', () => {
       .send(upgradeMessagePayload);
     expect(res).to.have.status(200);
 
-    let currentTag;
-    do {
-      currentTag = await upgradeService.getCurrentVersion(upgradeMessagePayload.containers[0].container_name);
-    } while (currentTag !== upgradeMessagePayload.containers[0].image_tag);
+    const imageTag = upgradeMessagePayload.containers[0].image_tag;
+    const containerPrefix = upgradeMessagePayload.containers[0].container_name;
+
+    await testContainerTag(upgradeService, `${containerPrefix}-1`, imageTag);
+    await testContainerTag(upgradeService, `${containerPrefix}-2`, imageTag);
   });
 
   it('Doesnt error if JSON format and container field has additional fields', async () => {
```

Comment:
Minor: This description could use an update as it's handling the additional fields rather than just not erroring. Also: Am I understanding it correctly that there's no remaining tests checking for failed upgrades? Not overly concerned by this as they're currently not handled as far as I know.

Reply:
This test still handles "failed" upgrades, just the output is changed, so we don't report on which container names weren't matched, which is fine, API doesn't expect any type of response when it calls the upgrade endpoint. 
It's not exactly a failed upgrade, because it's not a demand to absolutely match all containers in the payload.